test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestThreadMXBean.java

```diff
@@ -64,7 +64,7 @@ public class TestThreadMXBean {
 	private static Logger logger = Logger.getLogger(TestThreadMXBean.class);
 	private static final Map<String, AttributeData> attribs;
 	private static final HashSet<String> ignoredAttributes;
-	private static final boolean isIBMJava8 = System.getProperty("java.vm.vendor").contains("IBM");
+	private static final boolean isIBMJava8 = System.getProperty("java.vm.vendor").contains("IBM") && System.getProperty("java.version").contains("1.8.0");
```

Comment:
Suggest this should use `VersionCheck.major() == 8` instead of reading `java.version`.